Fix meta descriptions (#9399)
pantheon-systems/documentation: source/content/certification/study-guide-cms/03-create.md

---
title: WebOps Certification
subtitle: "Chapter 3: Site Creation"

> description: Understand and practice site creation on Pantheon.

Reviewer: ```suggestion
description: Learn how to create a site on Pantheon.
```
